critiquebrainz/frontend/views/test/test_moderators.py

```diff
@@ -0,0 +1,13 @@
+from flask import current_app
+from critiquebrainz.frontend.testing import FrontendTestCase
+
+
+class ModeratorsTestCase(FrontendTestCase):
+    def test_template_render(self):
+        try:
+            response = self.client.get('/moderators/')
+            self.assert200(response)
+            for admin in current_app.config['ADMINS']:
+                self.assertIn(admin, str(response.data.decode('utf-8')))
+        except:
```

Comment:
I am not sure. Maybe I should check for an empty list before it is used in sql request?